runtime/gc_base/RootScanner.hpp

```diff
@@ -434,7 +434,16 @@ class MM_RootScanner : public MM_BaseVirtual
 	 * which modifies elements within the list.
 	 */
 	virtual void scanOwnableSynchronizerObjects(MM_EnvironmentBase *env);
+	/**
+	 * scanContinuationObjects is for pruning continuation Objects, which are dead or last unmounted,
```

Comment:
terminology: I think we prune a list or remove object from list, but not prune objects
there are some other minor mistakes: missing space, missing letter, inconsistent casing...